examples/workflows/misc/daemon.py

```diff
@@ -36,6 +38,6 @@ def consumer(ip):
 
 with Workflow(generate_name="daemon-", entrypoint="d") as w:
     with DAG(name="d"):
-        s = server()
+        s: Task = server()
```

Comment:
why did you add this?

Reply:
So that intellisense knows `ip` exists on `s`. Otherwise:

![image](https://github.com/user-attachments/assets/022099df-7843-4c5c-877b-623dfb190957)

Reply:
I think this might be down to it selecting the "calling the original function" overload you're using when you call `server`, how does it look if you do `s = server(arguments={})` ?